_src/doxygen/doxygen.css

```diff
@@ -1802,7 +1801,7 @@ td.memTemplItemLeft div.hover
 
 table.memname, table.memname tr, table.memname td, table.memname th
 {
-  background: #202020 !important;
+  
 }
```

Comment:
We can just remove this whole block now I guess. :)

Reply:
Done